### DISCUSSION TITLE
Campo DateTime no  po-page-dynamic-edit

### DISCUSSION THREAD

**Discussion by RomuloSRamos** [po-ui/po-angular#2200] [Tire suas dúvidas]
No momento estou usando dois campos separados , junto no backend e salvo em uma coluna DATE do Oracle.
````
[

            {
                property: 'dataCmd',
                icon: "po-icon po-icon-calendar",
                label: 'Data/Hora do Comando',
                type: 'date',
                required: true,
                maskFormatModel: true,
                gridColumns: 2,
                gridSmColumns: 2,
                errorMessage: 'Data do Comando é obrigatória'
            },
            {
                property: 'horaCmd',
                label: 'Hora de comando',
                type: 'time',
                pattern: '^(0[0-9]|1[0-9]|2[0-3]):([0-5][0-9])$',
                required: true,
                gridColumns: 2,
                gridSmColumns: 2,
                errorMessage: 'Hora do comando deve ser uma hora válida'
            },


````
Eu não achei uma forma de fazer isso em um campo único tipo o do ionic [ion-datetime](https://ionicframework.com/docs/api/datetime)
<img width="240" alt="image" src="https://github.com/user-attachments/assets/c0351861-ea6c-4e1e-86b3-d8ae3397a58b"> <img width="280" alt="image" src="https://github.com/user-attachments/assets/010b870c-e29a-4fa9-8fe3-0e2572895f79">
Como é feito normalmente com PO.UI?

**Comment by ricardompb** [comment]
Pessoal, bom dia!

Existe alguma solução para esse caso???

Também estou com a mesma necessidade e não encontrei uma forma para tratar esse problema!

como eu guardo a data no meu banco de dados.

Ex: 
2025-06-17T00:00:00.000Z
